package.json

```diff
@@ -15,7 +15,7 @@
   "homepage": "https://github.com/cozy/cozy-ui",
   "scripts": {
     "build:css": "env CSSMODULES=false stylus -c --include stylus -o dist/cozy-ui.min.css stylus/cozy-ui/build.styl --use ./node_modules/autoprefixer-stylus --with \"{ browsers: ['last 2 versions'] }\"",
-    "build:css:app": "env CSSMODULES=false stylus --include stylus -o build/styleguide/app.css stylus/cozy-ui/build.styl --use ./node_modules/autoprefixer-stylus --with \"{ browsers: ['last 2 versions'] }\" --include-css",
+    "build:css:app": "env CSSMODULES=false stylus -c --include stylus -o build/styleguide/app.css stylus/cozy-ui/build.styl --use ./node_modules/autoprefixer-stylus --with \"{ browsers: ['last 2 versions'] }\"",
```

Comment:
`-c` is the new `--include css`?

Reply:
Not really no. 
`--include-css` allows to `@import` css file which we did until #833 There's no need anymore.
`-c` is for CSS compression.